Deploy/envirement.h

```diff
@@ -34,6 +34,13 @@ class DEPLOYSHARED_EXPORT Envirement
     void addEnv(const QString &dir);
     void addEnv(const QStringList &listDirs);
 
+    /**
+     * @brief isIgnore This method return true if the @a path is contains in ignore list else return false.
```

Comment:
What a symbol "@a"? Is this so necessary?

Reply:
this is argument backlight 
see the official documentation of the [doxygen](https://www.doxygen.nl/manual/commands.html#cmda)